### PR TITLE
final push

### PR DESCRIPTION
**Pull request by wyne-ybanez**
final push

### CODE REVIEW

File: templates/components/_modal.html [hunk 1]
```diff
@@ -1,43 +1,58 @@
...
-                
-                    Applicant: <span class="ps-3">{{ user.full_name }}</span> <br>
-                    Email: <span class="ps-3">{{ user.email }}</span> <br>
+<form class="col" method="POST" action="{{ url_for('job_applied', job_id=job._id) }}">
+    <div class="modal fade" id="job_apply" tabindex="-1" aria-labelledby="Job-Application-Modal" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h3 class="modal-title">Application for <span style="color: var(--s1-color);">{{ job.role }}</span> with 
+                        <span style="color: var(--s1-color);">{{ job.employer }}</span>
```
Comment (auxfuse) [inline templates/components/_modal.html:8]: Inline CSS use....can this be changed and identified in the external custom css file to maintain Separation of Concern Code principles?

**Review by auxfuse — CHANGES_REQUESTED** [review]
Left comments for review

File: templates/components/_modal.html (hunk 1 above)
Comment (auxfuse) [reply templates/components/_modal.html:8]: Yeah it's grand...

**Review by auxfuse — APPROVED** [review]
LGTM 🎉🎉🎉🎉🎉🎉🎉🎉

**Comment by wyne-ybanez** [comment]
Perfecto!!1

**Pull request merged**